experiment/contracts/BurnedBTC.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity >=0.4.21 <0.6.0;
+pragma experimental ABIEncoderV2;
+
+import {Crosschain} from "./Crosschain.sol";
+import {CheckpointRepo} from "./CheckpointRepo.sol";
+import {BurnVerifier} from "./BurnVerifier.sol";
+
+contract BurnedBTC is Crosschain {
```

Comment:
Do we want this to extend ERC20 also?

Reply:
Not for our experiments, but for actual deployments it could.